package.json

```diff
@@ -15,7 +15,8 @@
     "dev": "tsc --watch",
     "build": "tsc && tsc-alias",
     "e2e:plugin:publish": "cd cypress/fixtures/plugin && yarn && yarn livestorm publish",
-    "e2e:run:local": "yarn run e2e:plugin:publish && yarn cypress run --headless --browser chrome"
+    "e2e:run:local": "yarn run e2e:plugin:publish && yarn cypress run --headless --browser chrome",
+    "sass": "sass --watch --style=compressed src/assets/css:dist/assets/css"
```

Comment:
I would see the following changes:

- `build:css' (without the watch arg) and include it in the main `build` command
- add the `build:css` command with the watch arg in the `dev` command

What do you think ?

Reply:
Yes why not for the build command, but for the `dev` command it would require some changes since both command run as a server :/ I'll do it later I guess